.gitignore

```diff
@@ -1,3 +1,15 @@
-site/
-/venv
 /.vscode
+.idea/
+
+.DS_Store
+**/__pycache__/
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+
+# mkdocs documentation
+/site
```

Comment:
nit: This is a nitpick, and mostly commented for your future reference. Git, as a convention, mandates empty lines at the end of a file, otherwise you get the angry red circle like above.